### DISCUSSION TITLE
Permission error

### DISCUSSION THREAD

**Discussion by sahil13092000** [datasciencecompleteguide/Machine-Learning#14] [Q&A]
![IMG_20210708_230951](https://user-images.githubusercontent.com/76654411/124967233-ceaef280-e041-11eb-9304-75ba3f2b5c38.jpg)

I am getting error like this I have tried by administrator log in but still it is getting error please help me to resolve this issue......

**Comment by sahil13092000** [comment]
I have uploaded image again can you see it now

**Comment by datasciencecompleteguide** [comment]
> 
> 
> ![IMG_20210708_230951](https://user-images.githubusercontent.com/76654411/124967233-ceaef280-e041-11eb-9304-75ba3f2b5c38.jpg)
> 
> I am getting error like this I have tried by administrator log in but still it is getting error please help me to resolve this issue......


First Method:
Try running the cmd as "Run as admin."
if that also doesn't work then try:
Second Method:
install the package to the user folder
python -m pip install --user dlib
